app/controllers/api/v1/credentials_controller.rb

```diff
@@ -3,7 +3,16 @@ class CredentialsController < ApiController
     respond_to :json
```

Comment:
Does this method not blow up? The class method `respond_to` has been [removed from Rails][1]

[1]: http://api.rubyonrails.org/classes/ActionController/MimeResponds/ClassMethods.html#method-i-respond_to